Sources/Datadog/FeaturesIntegration/WebView/WKUserContentController+Datadog.swift

```diff
@@ -16,10 +16,32 @@ public extension WKUserContentController {
     internal func __addDatadogMessageHandler(allowedWebViewHosts: Set<String>, hostsSanitizer: HostsSanitizing) {
         let bridgeName = DatadogMessageHandler.name
 
+        let contextProvider = (Global.rum as? RUMMonitor)?.contextProvider
+
+        var logEventConsumer: WebLogEventConsumer? = nil
+        if let loggingFeature = LoggingFeature.instance {
+            logEventConsumer = WebLogEventConsumer(
+                userLogsWriter: loggingFeature.storage.writer,
+                internalLogsWriter: InternalMonitoringFeature.instance?.logsStorage.writer,
```

Comment:
`logsStorage.writer` is a [consent-aware data writer](https://github.com/DataDog/dd-sdk-ios/blob/fc693d3089d93ba3bc8e5d3ef21e841b6e973125/Sources/Datadog/Core/Feature.swift#L91). Do we need to consider tracking consent for events received from Browser SDK? Shouldn't we assume that all these were already authorized in Browser SDK and we can bypass consent consideration in native SDK?